pkg/webui/components/button/index.js

```diff
@@ -42,53 +43,38 @@ const assembleClassnames = ({
   primary,
   secondary,
   naked,
-  unstyled,
   grey,
   icon,
+  small,
   busy,
   dropdownItems,
   className,
   error,
-  disabled,
-}) => {
-  if (unstyled) {
-    return className
-  }
-  return classnames(style.button, className, {
+}) =>
+  classnames(style.button, className, {
     [style.danger]: danger,
     [style.warning]: warning,
     [style.primary]: primary,
     [style.secondary]: secondary,
     [style.naked]: naked,
     [style.busy]: busy,
     [style.grey]: grey,
+    [style.small]: small,
     [style.withIcon]: icon !== undefined && message,
     [style.onlyIcon]: icon !== undefined && !message,
     [style.withDropdown]: Boolean(dropdownItems),
     [style.error]: error && !busy,
-    [style.disabled]: disabled || busy,
   })
-}
 
 const buttonChildren = props => {
-  const {
-    dropdownItems,
-    icon,
-    busy,
-    message,
-    expanded,
-    noDropdownIcon,
-    dropdownClassName,
-    children,
-  } = props
+  const { dropdownItems, icon, busy, message, expanded, noDropdownIcon, children } = props
 
-  const content = Boolean(children) ? (
-    children
-  ) : (
+  const content = (
     <>
-      {icon ? <Icon className={style.icon} icon={icon} /> : null}
-      {message ? <Message content={message} className={style.linkButtonMessage} /> : null}
-      {dropdownItems ? (
+      {icon && <Icon className={style.icon} icon={icon} />}
+      {message && <Message content={message} className={style.linkButtonMessage} />}
+      {children}
```

Comment:
I don't think it's necessary to render either children or the dropdown markup. The children can always be rendered into the button regardless.